js/exporter/jspdf/v3/rows_splitting.js

```diff
@@ -0,0 +1,63 @@
+import { isDefined } from '../../../core/utils/type';
+
+function applySplitting(pdfCellsInfo, options) {
```

Comment:
applySplitting - let's rename it to splitToPages()

Reply:
Done